tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-initial-dialog.html

```diff
@@ -60,9 +76,19 @@ <h2 id='dialog-title'>Waiting for first Session.run() to connect...</h2>
         type: String,
         value: null,
       },
+      _open: {
```

Comment:
Can we just use the dialog's native `opened` property?
https://www.webcomponents.org/element/PolymerElements/paper-dialog/elements/paper-dialog#property-opened

Reply:
SG. Like this?